location/gql_mutations.py

```diff
@@ -94,10 +94,12 @@ class Input(LocationInputType):
 
     @classmethod
     def async_mutate(cls, user, **data):
-        if not Location.objects.filter(code=data['code'], type=data['type'], validity_to=None).exists():
```

Comment:
We can remove it and use differrent method, but the behaviour changes. check_unique_code does not care about type of the location, do we have any info about that? Should location code be unique across all locations or only given type?

Reply:
Good question, currently our approach is that the code is unique globally. Frontend validation is done in this way too.

Reply:
I would say yes it should be unique across all locations